src/rhsmlib/facts/collector.py

```diff
@@ -102,7 +108,7 @@ def get_all(self):
 
 
 class StaticFactsCollector(FactsCollector):
-    def __init__(self, static_facts, **kwargs):
+    def __init__(self, static_facts=DEFAULT_STATIC_FACTS, **kwargs):
```

Comment:
This worries me a little bit.  I worry that someone creates a dictionary of static facts and sends it in and that will mean that "system.certificate_version" will be undefined (unless the person remembers to define it in their dictionary).  Not having "system.certificate_version" set will mean the client will get version 1 entitlement certificates which is definitely not what we want.

How about instead:

```python
def __init__(self, static_facts=None, **kwargs):
  if static_facts is None:
    static_facts = {}
  static_facts.set_default("system.certificate_version, "3.2")
  ...
```

Or something similar.  Basically just something to insure that "system.certificate_version" is defined even if the dictionary `__init__` receives doesn't have it.